### ISSUE TITLE
Hangs sometimes when viewing the LED theme

### ISSUE THREAD

**Issue by algernon** [algernon/Chrysalis#59]
Not sure where the bug is, but I noticed the other day that viewing the (WIP) led theme page sometimes hangs the application so hard that it has to be restarted. Exact circumstances are not know yet.

**Comment by algernon** [comment]
At the moment, it looks like this happens when Chrysalis races for the control of the device... perhaps open blocks? Will have to strace, most likely.

**Comment by algernon** [comment]
Happens even after #58, though I had hoped that would fix it. But no :/

**Comment by algernon** [comment]
Further inspection shows that the renderer process is constantly `SIGSEGV`ing.

**Comment by algernon** [comment]
When this happens, I see the following in the terminal:

```
[28096:0508/115611.108365:ERROR:CONSOLE(0)] "null", source:  (0)
[28096:0508/115700.981776:ERROR:CONSOLE(0)] "null", source:  (0)
```

Appears to be a NULL deref somewhere.

**Comment by algernon** [comment]
Might be an issue with hot code reloading... I can't reproduce it when the code remains unchanged.

This means that this is not a critical bug at this time, and I may postpone fixing it until after the 0.0.2 release.

**Comment by algernon** [comment]
Can't reproduce after #76, closing.
